test/Performance/DynamoPerformanceTests/PerformanceTestFramework.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Validators;
+using Dynamo;
+
+namespace DynamoPerformanceTests
+{
+    public class AllowNonOptimized : ManualConfig
+    {
+        public AllowNonOptimized()
+        {
+            Add(JitOptimizationsValidator.DontFailOnError);
+
+            Add(DefaultConfig.Instance.GetLoggers().ToArray()); // manual config has no loggers by default
+            Add(DefaultConfig.Instance.GetExporters().ToArray()); // manual config has no exporters by default
+            Add(DefaultConfig.Instance.GetColumnProviders().ToArray());
+        }
+    }
+
+    //[Config(typeof(AllowNonOptimized))]
+    public class PerformanceTestFramework : DynamoModelTestBase
```

Comment:
@aparajit-pratap doesn't this mean this class now has a dependency on nunit 2.x?

Reply:
Nope. I verified this by removing dependencies on NUnit and it works. If this project still has a dependency on NUnit, I may have forgotten to clean it up during my experiments.